xc/common/utils/prjxray_routing_import.py

```diff
@@ -1302,7 +1311,23 @@ def main():
     populate_hclk_cmt_tiles(db)
 
     synth_tiles = None
-    if args.synth_tiles:
+    if args.synth_tiles and args.overlay:
```

Comment:
This should probably be "args.overlay" and then assert that args.synth_tiles is supplied.;